hazelcast/src/main/java/com/hazelcast/kubernetes/KubernetesApiEndpointResolver.java

```diff
@@ -190,4 +190,4 @@ void start() {
     void destroy() {
         client.destroy();
     }
-}
+}
```

Comment:
Why do you change the file end here?
This should be reverted - including other places with the same change.